src/crypto/mod.rs

```diff
@@ -457,13 +457,13 @@ where
         .map(|(i, sample)| (from_repr_plus_1::<C::Scalar>(i.clone().into()), sample))
         .collect();
     if samples.len() < t {
-        return Err(ErrorKind::NotEnoughShares.into());
+        return Err(Error::NotEnoughShares.into());
```

Comment:
This `into()` is probably redundant now?

Reply:
Yup, missed those.